compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -200,44 +240,63 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
             addCPUDefaultPassPipeline(executableLoweringPipeline);
             break;
           case IREE::Codegen::DispatchLoweringPassPipeline::
-              CPUBufferOpsTileAndVectorize:
-            addCPUBufferOpsTileAndVectorizePipeline(executableLoweringPipeline,
-                                                    enableVectorMasking,
-                                                    enableAArch64SSVE);
+              CPUBufferOpsTileAndVectorize: {
+            assert(succeeded(maybeLoweringConfig) && "Expected root operation");
+            TilingConfig tilingConfig(*maybeLoweringConfig);
```

Comment:
That was the first implementation but not all the pipelines require a lowering config so I can't create the TilingConfig outside of the switch. I can create a utility function with this two lines if that helps... Any other idea?

Reply:
I think we can allow empty TilingConfig as default. If the lowering_config is not provided, it basically returns empty lists for all the methods. Then it would be something like

```cpp
TilingConfig tilingConfig
if (succeeded(maybeLoweringConfig)) {
  tilingConfig.setConfig(*maybeLoweringConfig);
}
```

(IIUC, this can also address the failures in pipeline_tests.mlir.)